gensim/models/word2vec.py

```diff
@@ -675,14 +679,14 @@ def scan_vocab(self, sentences, progress_per=10000, trim_rule=None):
                         type(sentence)
                     )
                 checked_string_types += 1
-            if sentence_no % progress_per == 0:
+            if sentence_no % progress_per == 0 and sentence_no != 0:
```

Comment:
Why did this need?

Reply:
Because 0% anything will equal to 0; so the logger will log a statement saying sentence 0 and processed 0.

Reply:
But we want that :)

Reply:
But we want that :)